src/mlpack/core/dists/gamma_distribution_impl.hpp

```diff
@@ -10,25 +10,28 @@
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
+#ifndef _MLPACK_CORE_DISTRIBUTIONS_GAMMA_DISTRIBUTION_IMPL_HPP
+#define _MLPACK_CORE_DISTRIBUTIONS_GAMMA_DISTRIBUTION_IMPL_HPP
+
 #include "gamma_distribution.hpp"
 
-using namespace mlpack;
-using namespace mlpack::distribution;
+namespace mlpack {
+namespace distribution /** Probability distributions. */ {
 
-GammaDistribution::GammaDistribution(const size_t dimensionality)
+inline GammaDistribution::GammaDistribution(const size_t dimensionality)
 {
   // Initialize distribution.
   alpha.zeros(dimensionality);
   beta.zeros(dimensionality);
 }
 
-GammaDistribution::GammaDistribution(const arma::mat& data,
+inline GammaDistribution::GammaDistribution(const arma::mat& data,
                                      const double tol)
```

Comment:
```suggestion
inline GammaDistribution::GammaDistribution(const arma::mat& data,
                                                                 const double tol)
```

Reply:
Anyway, my suggestion is not good either, GitHub rendering is not that good, but I think  you got the idea